src/components/NavBar/SearchBar.css.ts

```diff
@@ -29,6 +29,7 @@ const baseSearchStyle = style([
 const baseSearchNftStyle = style([
   baseSearchStyle,
   {
+    borderWidth: '2px',
```

Comment:
why is this only for the nft one?

Reply:
I don't want to update main right now.  This is all behind phase 1 flag.  They should both use the same search once we go live